src/Files.App/ViewModels/UserControls/AddressToolbarViewModel.cs

```diff
@@ -225,6 +232,16 @@ private async void UpdateService_OnPropertyChanged(object? sender, PropertyChang
 				await CheckForReleaseNotesAsync();
 		}
 
+		private async void WindowContext_OnPropertyChanged(object? sender, PropertyChangedEventArgs e)
```

Comment:
Do we also need to watch for changes to settings?

Reply:
It is not in Settings dialog and it will never be changed runtime.
We don’t have a way to watch file change from outside yet but once done we should watch.

Reply:
In that case it's fine, I thought we were adding to Settings.